pkg/apis/minio.min.io/v2/helper.go

```diff
@@ -717,6 +782,23 @@ func (t *Tenant) CreateUsers(madmClnt *madmin.AdminClient, userCredentialSecrets
 	return nil
 }
 
+// CreateBuckets creates buckets and skips if bucket already present
+func (t *Tenant) CreateBuckets(minioClient *minio.Client, buckets []Bucket) error {
+	// create buckets with a 20 seconds timeout
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*20)
+	defer cancel()
+	for _, bucket := range buckets {
+
+		err := minioClient.MakeBucket(ctx, bucket.Name, minio.MakeBucketOptions{Region: bucket.Region, ObjectLocking: bucket.ObjectLocking})
+		if err != nil {
+			klog.Info("Bucket  creation failed %s error %s", bucket.Name, err.Error())
```

Comment:
```suggestion
			klog.Info("Bucket creation failed %s error %s", bucket.Name, err.Error())
```

Reply:
We should return the error if there is a `bucket` creation failure perhaps? @sinhaashish

Reply:
done